src/core/geometry/junction/Junction.cpp

```diff
@@ -0,0 +1,47 @@
+#include "Junction.h"
+#include <stdexcept>
+
+namespace semba::geometry::junction {
+	const std::string Junction::INVALID_ATTRIBUTE_MESSAGE = "Error: There must be at least 2 coordinates to unite!";
+
+
+	Junction::Junction(std::string & name)
```

Comment:
`const string&` is much better; gives warranties to the caller.

Reply:
Addressed in commit e0876b681447e7be6b8220111352cf1fac6c7c78.